0-common/scala-core/src/main/scala/com.snowplowanalytics.iglu/core/SchemaVer.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.iglu.core
+
+/**
+ * Class holding semantic version for Schema
+ *
+ * @param model Schema MODEL, representing independent Schema
+ * @param revision Schema REVISION, representing backward-incompatible changes
+ * @param addition Schema ADDITION, representing backward-compatible changes
+ */
+case class SchemaVer(model: Int, revision: Int, addition: Int) {
+  def asString = s"$model-$revision-$addition"
+}
+
+object SchemaVer {
+  /**
+   * Regular expression to validate SchemaVer
+   */
+  val schemaVerRegex = "\\d+-\\d+-\\d+".r
+
+  /**
+   * Regular expression to extract MODEL, REVISION, ADDITION
+   */
+  val modelRevisionAdditionRegex = "([0-9]+)-([0-9]+)-([0-9]+)".r
```

Comment:
Didn't we have a user somewhere suggest a tighter definition for SchemaVer? As `007/1/020` is not a valid version...